bson/src/test/unit/org/bson/codecs/pojo/PojoCustomTest.java

```diff
@@ -423,8 +422,7 @@ public void testEncodingInvalidCollectionModel() {
             encodesTo(getPojoCodecProviderBuilder(InvalidCollectionModel.class), new InvalidCollectionModel(asList(1, 2, 3)),
                 "{collectionField: [1, 2, 3]}");
         } catch (CodecConfigurationException e) {
-            assertTrue(e.getMessage().startsWith("Could not create a PojoCodec for 'InvalidCollectionModel'."
-                    + " Property 'collectionField' errored with:"));
+            assertTrue(e.getMessage().startsWith("Failed to encode 'InvalidCollectionModel'. Encoding 'collectionField' errored with:"));
```

Comment:
Mirrors the decoding errors: https://github.com/rozza/mongo-java-driver/blob/e9fd82a3825d6e985fc3d49bff05028b139341d3/bson/src/main/org/bson/codecs/pojo/PojoCodecImpl.java#L195